_automating-configurations/workflow-builder.md

```diff
@@ -0,0 +1,121 @@
+13. To view results, select **Search**. You can view the results as either a formatted list of hits or as the raw JSON search response.
+14. Adjust your configurations to better suit your use case. You can do this in several ways:
+   - Experiment with different query parameters.
+   - Try different queries.
+   - Modify existing processors under **Enhance query request** or **Enhance query response**.
+   - Add or remove processors under **Enhance query request** or **Enhance query response**.
+15. Export your workflow by selecting **Export** in the header to open the pop-up window. The displayed data represents the [Workflow Template]({{site.url}}{{site.baseurl}}/automating-configurations/workflow-templates/), which contains the full configuration for the OpenSearch resources you've just created, including the ingest pipeline, index, and search pipeline. You can download the template in JSON or YAML format by clicking the button on the right-hand side. To build identical resources in other OpenSearch clusters, use the [Provision Workflow API]({{site.url}}{{site.baseurl}}/automating-configurations/api/provision-workflow/).
+
+## Advanced data transformations
+
+ML inference processors provide several flexible ways to transform input data _to_ the model input(s) and _from_ the model output(s).
+
+In **Inputs**, you can configure the parameters passed _to_ the model. Each input parameter can be one of four transformation types:
+
+1. **Data field**: Use an existing field from your data as the model input.
+2. **JSONPath expression**: Extract data from a JSON structure and map the extracted data to the model input field using [JSONPath](https://en.wikipedia.org/wiki/JSONPath).
+3. **Prompt**: Use a constant value that can include dynamic variables. This combines elements of both the `Custom string` transformation and the `Data field` and `JSONPath expression` transformations, making it especially useful for building prompts for LLMs.
+4. **Custom string**: Use a constant string value.
+
+In **Outputs**, you can configure the values passed _from_ the model. Each output parameter can be one of three transformation types:
+
+1. **Data field**: Copy the model output into a new document field.
+2. **JSONPath Expression**: Extract data from a JSON structure and map the extracted data to one or more new document fields using [JSONPath](https://en.wikipedia.org/wiki/JSONPath).
+3. **No transformation**: Leave the model output field as is, preserving both its name and value.
+
+## Next steps
+
+- For suggested models and model interfaces to use within OpenSearch Flow, see [Models](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/documentation/models.md).
+
+- For additional tutorials on using OpenSearch Flow to build different AI/ML use cases, see [Tutorial](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/documentation/tutorial.md).
+
```

Comment:
The note I deleted would be more appropriate to a blog post as opposed to the documentation.